site/source/docs/api_reference/emscripten.h.rst

```diff
@@ -361,6 +361,8 @@ Functions
 
   .. note:: Calling this function overrides the effect of any previous calls to :c:func:`emscripten_set_main_loop_timing` in the calling thread by applying the timing mode specified by the parameter ``fps``. To specify a different timing mode for the current thread, call the function :c:func:`emscripten_set_main_loop_timing` after setting up the main loop.
 
+  .. note:: Currently, ``simulate_infinite_loop`` == true does not work yet with `the new Wasm exception handling <https://emscripten.org/docs/porting/exceptions.html#webassembly-exception-handling-proposal>`_; your program may crash with memory errors.
```

Comment:
Perhaps mention "does not work in C++ projects that have objects with destructors on the stack at the time of the call"?

Reply:
Done